Apps/Admin/Client/Pages/Support.razor

```diff
@@ -5,6 +5,12 @@
 <PageTitle>Support</PageTitle>
 <HgPageHeading>Support</HgPageHeading>
 
+<CascadingValue  Value = "@IsBannerVisible">
+    <MudHidden Invert="!HasError"> 
+    <HgBannerFeedback Variant="Variant.Text" Severity="Severity.Error" ShowCloseIcon="true" >@MessageVerificationState?.Value?.ErrorMessage</HgBannerFeedback>
+    </MudHidden>
+</CascadingValue>
```

Comment:
@MikeLyttle Why?

Reply:
MudHidden is for hiding things at certain breakpoints, rather than hiding things in general, and it resolves down to an `if` statement under the hood anyway, so it would be simpler to cut out the extra component.